data_structures/linked_list/reverse_linked_list_program.py

```diff
@@ -0,0 +1,55 @@
+# Time Complexity : O(n) 
+# Space Complexity : O(1) 
+
+# Node class 
+class Node: 
+
+	# Constructor to initialize the node object 
+	def __init__(self, data): 
+		self.data = data 
+		self.next = None
+
+class LinkedList: 
+
+	# Function to initialize head 
+	def __init__(self): 
+		self.head = None
+
+	# Function to reverse the linked list 
+	def reverse(self): 
+		prev = None
+		current = self.head 
+		while(current is not None): 
```

Comment:
```suggestion
		while current: 
```
If `current` variable is going to become `None` in the future, `while` loop will exit.